### DISCUSSION TITLE
milvus数据迁移

### DISCUSSION THREAD

**Discussion by zhangzf875** [milvus-io/milvus#27519] [Q&A and General discussion]
如果想要将qdrant的向量数据迁移到milvus，这个能实现吗，具体是要怎么操作

**Comment by zhangzf875** [comment]
请大佬帮忙解答一下

**Reply by yhmo** [reply]
主要看qdrant有没有提供接口把它的数据提取出来。类似于milvus的query()那样的接口。我翻了一下它的文档，似乎没有。

**Reply by xiaofan-luan** [reply]
数据不大的话，直接用Scroll points读出来，然后写进milvus就可以了
